www/docs/en/dev/guide/support/index.md

```diff
@@ -122,17 +122,6 @@ the core plugins, additional APIs are available via
         <td data-col="wp8"  class="y"></td>
     </tr>
 
-    <tr>
-        <th><a href="../../reference/cordova-plugin-contacts/">Contacts</a></th>
```

Comment:
Is this intentional for this PR?

Reply:
No. It is not. I forgot the earlier PR wasn't merged yet

Reply:
Seems good now